site/content/docs/5.0/customize/options.md

```diff
@@ -24,6 +24,6 @@ You can find and customize these variables for key global options in Bootstrap's
 | `$enable-rfs`                  | `true` (default) or `false`        | Globally enables [RFS]({{< docsref "/getting-started/rfs" >}}). |
 | `$enable-validation-icons`     | `true` (default) or `false`        | Enables `background-image` icons within textual inputs and some custom forms for validation states. |
 | `$enable-negative-margins`     | `true` or `false` (default)        | Enables the generation of [negative margin utilities]({{< docsref "/utilities/spacing#negative-margin" >}}). |
-| `$enable-deprecation-messages` | `true` or `false` (default)        | Set to `true` to show warnings when using any of the deprecated mixins and functions that are planned to be removed in `v5`. |
+| `$enable-deprecation-messages` | `true` (default) or `false`        | Set to `true` to show warnings when using any of the deprecated mixins and functions that are planned to be removed in `v5`. |
```

Comment:
Should change `v5` to `v6` here, I guess?

Reply:
Of course @ffoodd, but only if there are any deprecation messages to be actually displayed.

Reply:
Shouldn't we inverse the description? Set to `false` to hide...

Reply:
So would this be correct?

```suggestion
| `$enable-deprecation-messages` | `true` (default) or `false`        | Set to `false` to hide warnings when using any of the deprecated mixins and functions that are planned to be removed in `v6`. |
```

Reply:
@ffoodd, looks great, for me 🙂